src/submission/parseErrors/index.jsx

```diff
@@ -86,12 +86,10 @@ class ParseErrors extends Component {
 
   render() {
     const props = this.props
+
     if (!props.fetched) return <Loading />
 
-    const total =
```

Comment:
We have this total already ... plus I was seeing it as off by minus 1. Let me know if you see this differently.